css/main.css

```diff
@@ -1,19 +1,158 @@
+@font-face {
+  font-family: ProximaNova;
+  src: url(../fonts/ProximaNova-Reg.ttf);
+}
+
 .hero-section {
-    background-image: url(../images/bg-hero.jpg);
-    background-size: cover;
-    min-height: 90vh;
+  background-image: url(../images/bg-hero.jpg);
+  background-size: cover;
+  min-height: 95vh;
 }
 .navbar-light .navbar-nav .nav-link {
-    color: white;
+  color: white;
 }
 .navbar-light .navbar-nav .nav-link:hover {
-    color: white;
-    text-decoration: underline;
+  color: white;
+  text-decoration: underline;
 }
 .main-form {
-    padding: 50px;
-    width: 400px;
-    background-color: rgba(255, 255, 255, 0.8);
-    border: solid 1px #d4d4d4;
+  padding: 35px;
+  width: 400px;
+  background-color: rgba(255, 255, 255, 0.8);
+  border: solid 1px #d4d4d4;
 
-}
\ No newline at end of file
+}
+#cta-button{
+border-radius: 3px;
+background-color: #cc5054;
+box-shadow: 0 3px 0 0 #940308;
+border: solid 1px #cc5054;
+}
+.info {
+  font-family: ProximaNova;
+  font-size: 18px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.33;
+  letter-spacing: normal;
+  text-align: left;
+  color: #333333;
+}
+
+.bioPic {
```

Comment:
Use kebab case for `CSS` names instead of camel case.
Example: change `. bioPic` -> `.bio-pic`